package-lock.json

```diff
@@ -1418,7 +1418,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
back and forth 🤔

Reply:
6.4.1, right? So this is a mystery...

Reply:
6.4.1... correct. Super odd.

Reply:
Looks like this is a known issue:

https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818
https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285

The question is why I get HTTPS then? 🤔🤔🤔🤔🤔